Contents/Code/__init__.py

```diff
@@ -190,13 +190,23 @@ def search(self, results, media, lang, manual):
         )
 
         info = self.run_search(search_helper, result)
+        
+        # Set localized "by"
+        by_lang_dict = {
+            Locale.Language.English: 'by',
+            'de': 'von',
+            'fr': 'de',
+            'it': 'di'
+        }
+        localized_sep = by_lang_dict.get(lang, "by")
+        log.debug("Localized separator between title and artist: " + localized_sep)
 
         # Output the final results.
         log.separator(log_level="debug")
         log.debug('Final result:')
         for i, r in enumerate(info):
-            description = '\"%s\" by %s [%s]' % (
-                r['title'], r['artist'], r['year']
+            description = '[%s] \"%s\" %s %s' % (
```

Comment:
@seanap what do you think about this style change?

Reply:
It looks like the year starts to be truncated at 81 characters. Alternatives would be to:

1. Truncate the search result text ourselves.
2. Only show year first if the `description` is 81 characters or more.

The issue is that with the year in front, the author is now more than likely going to be truncated. So to me, it seems this is a game of picking which data is most important.

Reply:
For me it starts truncating at 60 characters.
![grafik](https://user-images.githubusercontent.com/16120882/133273827-6cb3708a-a616-47d2-bdf0-fcc9d4bdb541.png)
Is it possible to add a column like there is for the score? That way the year can have a fixed spot at the end and doesn't bounce back an forth.

Reply:
Or is it possible to change font styles? Then we could leave the year at the front with a greyish color and leave out the brackets (saving 2 chars)

Reply:
Having a max length for author and/or title does sound like a good idea though. Only question is where to draw the line.